scripts/bootstrap.ps1

```
@@ -0,0 +1,126 @@
+Write-Host "Creating Storage Container: $CONTAINER_NAME..."
+$containerResult = az storage container create --name $CONTAINER_NAME --account-name $SA_NAME 2>&1
+if ($LASTEXITCODE -ne 0) {
+    $showResult = az storage container show --name $CONTAINER_NAME --account-name $SA_NAME 2>&1
+    if ($LASTEXITCODE -eq 0) { Write-Host "Container $CONTAINER_NAME already exists, reusing." }
+    else { Write-Error "Failed to create storage container $CONTAINER_NAME."; exit 1 }
+}
+
+$EXISTING_APP = az ad app list --display-name $APP_NAME --query "[0].appId" -o tsv 2>$null
+if ($EXISTING_APP) {
+    $APP_ID = $EXISTING_APP
+    Write-Host "Reusing existing Azure AD Application: $APP_NAME ($APP_ID)"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'bootstrap.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)